joladnijo/settings.py

```diff
@@ -76,9 +77,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.mysql',
-        'NAME': 'joladnijo',
-        'USER': 'joladnijo',
+        'ENGINE': os.environ.get('DBENGINE', ''),
```

Comment:
Szerintem ez maradhat a mysql erteken, mert ezt kulonbozo kornyezetekben ugysem fogjuk felulirni a kornyezeti valtozokbol.
```suggestion
        'ENGINE': 'django.db.backends.mysql',
```

Reply:
Mi a helyzet a tesztekkel? Azok is MySQL-lel futnak, a valódi adatbázison?

Reply:
Sőt, itt a CI pipeline-ban a migrationöknek se a rendes DB-n kellene futni, csak valami eldobhatón.

Reply:
Igen, minden kornyezetben MySQL 5.7 van, nehogy az legyen, hogy valami nem tamogatott feature-t kezdunk el hasznalni, es kulonbozo eredmeny legyen kulonbozo kornyezetben.